container-chains/primitives/authorities-noting-inherent/src/mock.rs

```diff
@@ -68,7 +71,11 @@ impl InherentDataProvider for MockAuthoritiesNotingInherentDataProvider {
         let assignment = AuthorityAssignmentSproofBuilder::<NimbusId> {
             authority_assignment: AssignedCollators {
                 orchestrator_chain: vec![],
-                container_chains: Default::default(),
+                container_chains: {
+                    let mut chains = BTreeMap::new();
+                    chains.insert(self.container_para_id, self.authorities.clone());
+                    chains
```

Comment:
This can be done as a one-liner using `BTreeMap::from_iter([(self.container_para_id, self.authorities.clone())])`, but I prefer it to be declared above using `let container_chains = ...`